.github/workflows/make.yml

```diff
@@ -0,0 +1,26 @@
+name: Build and Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run-tests:
+    name: Run tests
+    runs-on: ubuntu-latest
```

Comment:
Are we able to select a specific version? We need the pipeline to be as stable as possible. We don't want it to break if ubuntu decides to release a new version with a new compiler. I think we should use the image "ubuntu-22.04" and update it manually in the future if needed.

Reply:
yes we are able to select a version